Content.Tests/DMProject/Tests/Builtins/text2ascii_char.dm

```diff
@@ -1,2 +1,3 @@
 ﻿/proc/RunTest()
 	ASSERT(text2ascii_char("abcdéfg",6) == 102) //102 is f
+	ASSERT(ascii2text(text2ascii("a")) == "a")
```

Comment:
This file is for testing `text2ascii_char()`. Would be reasonable to rename this file to `text2ascii.dm` and use both forms in it though.

Would also be good to test some non-ASCII values.